<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenShakeDialog</class>
 <widget class="QWidget" name="ScreenShakeDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Screen Shake</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QSlider" name="slider_power">
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Time:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="slider_speed">
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spin_time">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Power:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Speed:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spin_power">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spin_speed">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Frames</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>slider_power</tabstop>
  <tabstop>spin_power</tabstop>
  <tabstop>slider_speed</tabstop>
  <tabstop>spin_speed</tabstop>
  <tabstop>spin_time</tabstop>
  <tabstop>button_cancel</tabstop>
  <tabstop>button_ok</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>slider_power</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_power</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_speed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_speed</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_power</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_power</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_speed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_speed</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
